src/getContext.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import * as React from 'react'
+import type { FormApi, FormValuesShape } from 'final-form'
+
+let instance: React.Context<?FormApi<any>>
+
+export default function getContext<
```

Comment:
@erikras what's the reason behind this? kinda surprising, is this lazy evaluation needed for anything?

Reply:
It's not about lazy, it's about typing. I couldn't figure a way to handle the typing.

I suppose I could've just cast the thing...